### DISCUSSION TITLE
Running DAG manually getting SubDAGs scheduled

### DISCUSSION THREAD

**Discussion by mihaly-hazag** [apache/airflow#30783] [Q&A] [answered]
I am migrating old Airflow code to 2.5.1

In one of the DAGs there are SubDAGs used. 

If I run the DAG manually the Run Id starts with "manual_" for the main DAG (which is OK), but the 5 SubDAGs' Run Id starts with "scheduled_". Can someone help me why does it happen and what I should change to get the proper Run id for the Subdags as well?
(as you can see on the picture below the datetimes in the Run ids after the prefix are identical)

<img width="114" alt="image" src="https://user-images.githubusercontent.com/12548302/233510424-52026f4b-26fd-4a5e-a61e-b2cb2651d99d.png">

**Comment by potiuk** [comment, marked as answer]
SUBDAGS Are depracated. Stop using them. Seriously. You can use Trigering other dags or Dataset driven scheduling now that you miggrate to 2.5 instead/

**Reply by mihaly-hazag** [reply]
Thank you, I will rewrite the DAG then.
